src/dependencies.jl

```diff
@@ -545,7 +545,7 @@ function _find_library(dep::LibraryDependency; provider = Any)
         end
         @linux_only begin
             soname = ccall(:jl_lookup_soname, Ptr{UInt8}, (Ptr{UInt8}, Csize_t), lib, sizeof(lib))
-            soname != C_NULL && check_path!(ret,dep,bytestring(soname))
+            soname != C_NULL && check_path!(ret,dep,unsafe_string(soname))
```

Comment:
could you rebase? I think this is the only one that hasn't been gotten elsewhere